config_test_native_sql.go

```diff
@@ -29,3 +33,27 @@ func (d *DefaultDatabaseBuilder) getGraphStore(baseDB *sqldb.BaseDB,
 		graphExecutor, opts...,
 	)
 }
+
+// graphSQLMigration is the version number for the graph migration
+// that migrates the KV graph to the native SQL schema.
+const graphSQLMigration = 9
+
+// getSQLMigration returns a migration function for the given version.
+func getSQLMigration(ctx context.Context, version int,
+	kvBackend kvdb.Backend,
+	chain chainhash.Hash) (func(tx *sqlc.Queries) error, bool) {
+
+	if version != graphSQLMigration {
```

Comment:
nit: formulate this as a `switch` that matches on version and returns the correct callback? That way it's more obvious to future migrations that they belong in here as well?